### PR TITLE
fix: more robust context menu tests

### PR DESCRIPTION
**Pull request by erickzhao**
Makes all context menu tests go green.

Improvements:
* Selectors grouped into a single object to organize them better.
* Selectors changed to depend less on CSS classes.
* Change tests to only use the `old-joe` workspace.

### CODE REVIEW

**Comment by erickzhao** [comment]
Realized there's overlap between this and #73

**Comment by loc** [comment]
This is a solid change, even though there's some overlap. I just landed #73, reopening this.

File: src/smoke/menu-context.ts [hunk 1]
```diff
@@ -50,6 +51,7 @@ export const test: SuiteMethod = async ({ it, beforeAll, beforeEach }) => {
     clipboard.writeText('beforeEach');
     centerMouse();
     await focus();
+    await switchToTeam(0);
```
Comment (loc) [inline src/smoke/menu-context.ts]: Should this be in `beforeAll()`?

Comment (erickzhao) [reply src/smoke/menu-context.ts]: @loc Yeah, we could put it there, now that I think about it.

**Pull request merged**